src/reactapp/src/context/Cart/cartReducer.js

```diff
@@ -22,6 +30,8 @@ const actions = {
   [SET_CART_BILLING_ADDRESS]: setCartBillingAddress,
   [SET_CART_SELECTED_SHIPPING_ADDRESS]: setCartSelectedShippingAddress,
   [SET_ORDER_INFO]: setOrderInfo,
+  [GET_CHECKOUT_AGREEMENTS]: getCheckoutAgreements,
```

Comment:
these method name and code is not apt. we are using this reduce method to SET agreements into cart global state.  So `[SET_CHECKOUT_AGREEMENTS]: setCheckoutAgreements,` would be more apt